File: python_project/Strings/AlphaNumaricCounter.py
s1 = "Abhi 2026!"
letterCount = 0
digitcount = 0
symboleCount = 0
for i in range (0, len(s1)):
    x = s1[i]
    if x.isalpha():
        letterCount = letterCount + 1
    elif x.isdigit():
        digitcount = digitcount + 1
    elif x.isspace():
        pass
    else:
        symboleCount = symboleCount + 1

print("Original Text: \"" + s1 + "\"")
print("-------------------------")
print("Letters count: " + str(letterCount))
print("Digits count : " + str(digitcount))
print("Symbols count: " + str(symboleCount))

File: java_project/Strings/AlphaNumaricCounter.java
package String;

public class AlphaNumaricCounter {
    public static void main(String[] args) {
        String s1 = "Abhi 2026!";
        int letterCount = 0;
        int digitcount = 0;
        int symboleCount = 0;
        //String s3 = " ";
        for (int i=0; i<s1.length(); i++){
            char x = s1.charAt(i);
            if (Character.isLetter(x)){
                letterCount++;
            }
            else if (Character.isDigit(x)){
                digitcount++;

            }
            else if (Character.isWhitespace(x) ){
                
            }
            else {
                symboleCount++;
            }
        }
        System.out.println("Original Text: \"" + s1 + "\"");
        System.out.println("-------------------------");
        System.out.println("Letters count: " + letterCount);
        System.out.println("Digits count : " + digitcount);
        System.out.println("Symbols count: " + symboleCount);
    }
}
